code/__DEFINES/~skyrat_defines/cargo_import_companies.dm

```diff
@@ -0,0 +1,28 @@
+// Bitflags for what company a cargo order datum should belong to
+#define CARGO_COMPANY_NAKAMURA_MODSUITS (1<<0)
+#define CARGO_COMPANY_BLACKSTEEL (1<<1)
+#define CARGO_COMPANY_NRI_SURPLUS (1<<2)
+#define CARGO_COMPANY_DEFOREST (1<<3)
+#define CARGO_COMPANY_DONK (1<<4)
+#define CARGO_COMPANY_KAHRAMAN (1<<5)
+#define CARGO_COMPANY_NAKAMURA_TOOLING (1<<6)
+#define CARGO_COMPANY_BOLT_NANOTRASEN (1<<7)
+#define CARGO_COMPANY_MICROSTAR (1<<8)
+#define CARGO_COMPANY_VITEZSTVI_AMMO (1<<9)
+
+// Company names, because the armament category and company name need to be the exact same, so use defines like this
+#define NAKAMURA_ENGINEERING_MODSUITS_NAME "Nakamura Engineering MOD Divison"
+#define BLACKSTEEL_FOUNDATION_NAME "Jarnsmiour Blacksteel Foundation"
+#define NRI_SURPLUS_COMPANY_NAME "Nizhny Company Military Surplus"
```

Comment:
Unless intentional, this basically literally translates as "The Lower Company", which doesn't make much sense tbh.